public/components/common/custom-search-bar/custom-search-bar.tsx

```diff
@@ -18,7 +18,7 @@ import { getCustomValueSuggestion } from '../../../components/overview/office-pa
 
 export const CustomSearchBar = ({ filtersValues, ...props }) => {
   const { filterManager, filters } = useFilterManager();
-  const indexPattern = useIndexPattern();
+  const filterDrillDownValue = sessionStorage.getItem('filterDrillDownValue')
```

Comment:
**issue**: Could we get a solution without using the `sessionStorage`?

**suggestion**: Maybe we should use some component `prop` to control when the selector is disabled.